src/Bundler.js

```diff
@@ -377,24 +378,33 @@ class Bundler extends EventEmitter {
     return asset;
   }
 
-  watch(path, asset) {
+  async watch(path, asset) {
     if (!this.watcher) {
       return;
     }
 
-    if (!this.watchedAssets.has(path)) {
-      this.watcher.watch(path);
-      this.watchedAssets.set(path, new Set());
-    }
+    if (await fs.exists(path)) {
+      if ((await fs.lstat(path)).isSymbolicLink()) {
```

Comment:
We should probably put `watch` and `unwatch` in a noop `try`/`catch` clause in case an error is thrown (eg. file locked by the anti-virus on Windows) to prevent unhandled promise rejections.

Reply:
Yeah was thinking the same but than thought people would open issues for some failed watches.

Reply:
What about just skipping (await fs.lstat(path)).isSymbolicLink() and just using await fs.realpath(path) instead? As it returns the original path if it's not a symbolic link? Seems to be faster as well

Reply:
👍 that's even better!